.github/workflows/conformance-gke-v1.10.yaml

```diff
@@ -193,7 +193,7 @@ jobs:
             --machine-type e2-custom-2-4096 \
             --disk-type pd-standard \
             --disk-size 10GB \
-            --node-taints node.cilium.io/agent-not-ready=true:NoExecute \
+            --node-taints ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready=true:NoExecute \
```

Comment:
Since we're doing this, I believe we also need this in this file:

```
--helm-set=agentNotReadyTaintKey=ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready \
```

Otherwise, I guess the taint won't be removed.

Reply:
Changed. Wondering why we don't need the other `--helm-set` values in `job installation-and-connectivity` like conformance-gke.yaml in v1.10 and v1.11 files?